src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,35 +6,78 @@ cloud: true
 
-## Authorize the right network (Dedicated)
+## SQL Best Practices
 
-CockroachDB {{ site.data.products.dedicated }} requires you to authorize the networks that can access the cluster to prevent denial-of-service and brute force password attacks. During the application development phase, you might have authorized only your local machine’s network. To move into production, you need to authorize your application server’s network.
+To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
 
-To verify that you have authorized the application server's network, navigate to the [**Networking** page]({% link cockroachcloud/connect-to-your-cluster.md %}#authorize-your-network) on the CockroachDB {{ site.data.products.cloud }} Console and check if you see the application server network in the list of authorized networks. If you do not see the application server network in the list, [authorize the network]({% link cockroachcloud/connect-to-your-cluster.md %}#authorize-your-network).
+## Use a pool of persistent connections
 
-## Configure PCI ready features (Dedicated advanced)
+Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
+
+For guidance on sizing, validating, and using connection pools with CockroachDB, see [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
+
+## Keeping connections current
+
+After an application establishes a connection to CockroachDB {{ site.data.products.cloud }}, those connections can occasionally become invalid. This could be due to changes in the cluster topography, rolling [upgrades]({% link cockroachcloud/upgrade-policy.md %}) and restarts, network disruptions, or cloud infrastructure unavailability.
+
+Make sure connection validation and retry logic is used by your application. Validating and retrying connections is typically handled by the driver, framework, or the connection pool used by an application. For guidance on connection pool sizing, connection validation, and connection retry logic, see [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
+
+## PCI ready features (Dedicated advanced)
```

Comment:
How did you decide how to order the checklist? Tiny nit but it's confusing if everything below is Dedicated Advanced or just this. It's odd having it in the middle for that reason but there might be other reasons to keep it here. I don't feel strongly -- just one viewpoint.

Reply:
There is a typo in one of the links (think this pre-dates these changes)
I believe "SClusterQL audit log export" should read "Cluster SQL audit log export"

Reply:
@mwang1026 The checklist follows the same order as the [Self-Hosted checklist](https://www.cockroachlabs.com/docs/v23.1/recommended-production-settings). I'm open to changing the structure tho.